packages/nextjs/app/builders/0xe9Ad7D1C2069E0Fa9b5852Adc77C9196651BB8b8/page.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import Image from "next/image";
+import Socials from "./components/Socials";
+import type { NextPage } from "next";
+import { useEnsAvatar, useEnsName } from "wagmi";
+import { Address } from "~~/components/scaffold-eth";
+
+const builderAddress = "0xe9Ad7D1C2069E0Fa9b5852Adc77C9196651BB8b8";
+const shortBuilderAddress = builderAddress.slice(0, 6);
+
+const KrvvsProfile: NextPage = () => {
+  const { data: fetchedEns } = useEnsName({
+    address: builderAddress,
+    chainId: 1,
+  });
+
+  const { data: fetchedEnsAvatar } = useEnsAvatar({
+    name: fetchedEns ? fetchedEns : "",
+    chainId: 1,
+  });
+
+  return (
+    <div className="flex justify-center">
+      <div className="w-[800px] min-w-[200px] max-w-[800px] p-[30px] m-[50px] bg-base-200 shadow-2xl rounded-2xl flex flex-col gap-6 items-center">
+        <div className="avatar">
+          <div className="w-40 rounded-full">
+            <Image
+              src={fetchedEnsAvatar || "https://avatars.githubusercontent.com/u/141290516?v=4"}
```

Comment:
I tested several addresses with ENS profiles, and noticed that the domain of the fetched avatar image URL varies from address to address. For the current builder address without an ENS profile, it falls back to a default, which works for now but could become an issue in the future.

Updating `next.config.js` to allow any domain on the allowlist doesn't seem like a safe approach. 

@phipsae Do you have any suggestions on how to handle this more securely?